### DISCUSSION TITLE
Pi-Hole

### DISCUSSION THREAD

**Discussion by Mephisto090** [mcuiobroker/docuJarvis#6] [Ideas]
Hej,

hast du schon ein schönes Tutorial zu "schönen" vis von Pi-Hole?

**Comment by mcuiobroker** [comment]
Nein. Das Thema steht bei mir auch erst mal ganz weit hinten.

**Reply by Mephisto090** [reply]
Na ich versuche da mal was.

**Reply by Mephisto090** [reply]
Das Bar-Chart kommt nicht mit einem json-Format klar oder?

```
[
  {
    "1664714100": 0,
    "1664714700": 11,
    "1664715300": 14,
    "1664715900": 14
  }
]
```

**Reply by mcuiobroker** [reply]
https://github.com/Zefau/ioBroker.jarvis/issues/1410

**Reply by Mephisto090** [reply]
ich sehe da immer noch nicht durch. Ist das jetzt eingebaut, wird das noch eingebaut oder muss man noch dafür Voten ob es eingebaut wird? :D

**Reply by mcuiobroker** [reply]
Zurzeit muss man voten, damit es einen höheren Stellenwert bekommt und er sich das dann näher anschaut.
